CONTRIBUTING.md

```diff
@@ -0,0 +1,23 @@
+# Contribution
+
+In order to contribute to the improvement of the project, below you will find the project development flow.
+Before being able to open a PR with changes to be applied or bug fixes to be solved, it is necessary to open an issue 
+and explain the reason and a possible solution. You can open each PR by forking the project and creating the PR directly from 
+your project to the project of our library.
```

Comment:
same as for the JS library. We should add a sentence or two regarding our unit tests.